### DISCUSSION TITLE
Contacting sponsors

### DISCUSSION THREAD

**Discussion by magenta404** [community/community#3930] [Other Feature Feedback, Questions, & Ideas]
When someone subscribes, we only get a username. Rewards for sponsorship often require contacting the sponsor to setup. Is there, or can there be, a reliable way to contact sponsors?

It would also be helpful if we can customise what sponsors get after sponsorship. Then we could provide instructions.

**Comment by bdresser** [comment]
Hey @daleeidd! We're working on a feature that lets you customize the welcome message folks see after Sponsoring you. You'll be able to customize this message per tier. 

Feel free to email me at bdresser@github.com if you'd like to do some usability testing with an early prototype.

**Reply by magenta404** [reply]
Sure. Emailed.

**Reply by bdresser** [reply]
Hey @daleeidd – this feature has actually made it all the way to production :) 

On the tier builder screen, there's now a text area for each tier that lets you define a "welcome message" to folks who sponsor you at that tier. This message will show on the confirmation screen shown after checkout, and it will also be included in the email receipt your sponsors receive. We'll be posting more documentation about this in the coming days.

Give it a spin and let us know if you have any feedback!

**Reply by KaelWD** [reply]
@bdresser The "send test email" button doesn't seem to work for me, it just sends me back to the tiers list and says "You've updated a tier" but I don't get an email. I have checked my spam folder and there's nothing in there either.
